examples/features/vl3-lb/finance-lb-patch.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vl3-lb
+spec:
+  template:
+    metadata:
+      labels:
+        type: vl3-client
+    spec:
+      containers:
+        - name: lb
+          env:
+            - name: POD_NAME
+              value: "finance"
```

Comment:
Why do we need a hardcoded pod name env here?

Reply:
We use POD_NAME as DNS name in vl3 examples

Reply:
I think it's worth mentioning in the readme file or adding a comment in the yaml file.
So that if someone wants to replicate vl3-lb setup it's clear what they should change.

I think, ideally there should be a different solution.
Each pod gets `<pod-name>.vl3-domain` address, and additionally some pods may get more DNS addresses.